raiden_libs/messages/paths_reply.py

```diff
@@ -0,0 +1,107 @@
+# -*- coding: utf-8 -*-
+from typing import Dict
+
+import jsonschema
+from eth_utils import is_address, decode_hex
+
+from raiden_libs.messages.message import Message
+from raiden_libs.properties import address_property
+from raiden_libs.messages.json_schema import PATHS_REPLY_SCHEMA
+from raiden_libs.utils import eth_verify, UINT256_MAX, pack_data
+from raiden_libs.types import Address
+
+
+class PathsReply(Message):
+    """ A message to request a path from PFS. It is sent from a raiden node to the PFS. """
+    def __init__(
+        self,
+        token_network_address: Address,
+        target_address: Address,
```

Comment:
General question: do we need a starter_address as well?

Reply:
I suppressed it, as the requesting client will know its own address.

Reply:
In that use case sure, but maybe you want to query for some other node...

Reply:
Sure potentially, but than again this node will know its own address. I wouldn't add tautological information, especially for privacy reasons.

Reply:
The information is in the `paths` array anyways.

Reply:
...making it even more redundant to add.

Reply:
But adding convenience for the client. For me your argument falls down when you still send back value, chain_id, token_network and num_paths. That's all stuff that's known by the client.

Reply:
Follow up:  `value, chain_id, token_network` and `num_paths` are in fact necessary if the client requests paths in different `token_networks` at the same time, which is possible. The PFS responses may arrive asynchronously, so the client needs this information in order to determine on which token network the paths are located.